cmd/rhoas/main.go

```diff
@@ -143,7 +141,7 @@ func moveConfigFile(cfg config.IConfig) error {
 	}
 	// create rhoas config directory
 	if _, err = os.Stat(rhoasCfgDir); os.IsNotExist(err) {
-		err = os.MkdirAll(rhoasCfgDir, 0700)
```

Comment:
Any unknown implications of this?

Reply:
I was also really puzzled by it but after reading it makes sense as this is the official bit flag defined in the os api. That comes from formatter auto changes and it is not result of any rule processing.